spec/rails_helper.rb

```diff
@@ -49,3 +49,7 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+Capybara.register_driver :rack_test do |app|
```

Comment:
Separate commit?

Reply:
Could do, it is mainly connected to these changes though?

Reply:
This change is changing the way Capybara works so I'm guessing is general config change and not specific to the bug fix? Unless I'm mistaken.

Reply:
Done, split it out!